README.md

```diff
@@ -197,6 +204,41 @@ To uninstall the module, delete the following subtrees:
 * /apps/saas-aem-module-packages
 * /home/users/system/saas
 
+## Getting started
+
+After installation of the module following steps need to be done:
+
+### OSGI Configurations
+* Configure [Search as a Service - Search Api Request Executor Service Configuration](http://localhost:4502/system/console/configMgr/com.valtech.aem.saas.core.http.client.DefaultSearchApiRequestExecutorService):
+  * Base URL
+  * JWT Token
+* Configure [Search as a Service - Search Admin Request Executor Service Configuration](http://localhost:4502/system/console/configMgr/com.valtech.aem.saas.core.http.client.DefaultSearchAdminRequestExecutorService)
+  * Base URL
+  * JWT Token or Basic Authentication 
+
+Within AEM as a Cloud Service Base URL and JWT Token can be configured as a [secret environment variable](https://experienceleague.adobe.com/docs/experience-manager-cloud-service/content/implementing/deploying/configuring-osgi.html?lang=en#secret-configuration-values) 
+
+### Context-aware configurations
+
+The following fields are mandatory: SaaS Index
```

Comment:
Where do I find the context-aware configurations?
(asking because I didn't have to set one up for my env, as far as I can tell 🤔 )

Reply:
yes, there is a default index already configured in https://github.com/valtech-ch/saas-aem-module/blob/main/ui.content/src/main/content/jcr_root/conf/saas-aem-module/_sling_configs/com.valtech.aem.saas.api.caconfig.SearchConfiguration/.content.xml#L8

This is referenced by https://github.com/valtech-ch/saas-aem-module/blob/main/ui.content/src/main/content/jcr_root/content/saas-aem-module/.content.xml#L11 , so when using page in /content/saas-aem-module it is already preconfigured